e2e/WORKSPACE

```diff
@@ -0,0 +1,74 @@
+workspace(name = "my_workspace")
+
+load("@bazel_tools//tools/build_defs/repo:local.bzl", "local_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
+
+local_repository(
+    name = "cloud_archive",
+    path = "..",
+)
+
+load("@cloud_archive//:cloud_archive.bzl", "minio_archive", "minio_file")
+
+minio_archive(
+    name = "test_archive",
+    build_file = "//:BUILD.archive",
+    file_path = "local/bucket/data.tar.gz",
+    sha256 = "cb8c43418acb05e11619478fbb9c0a76827841cc2d1f58ce9b4d7b5093d1c01e",
+)
+
+minio_archive(
+    name = "test_archive_p0_patch",
+    patches = ["@//:patches/p0.patch"],
+    build_file = "//:BUILD.archive",
+    file_path = "local/bucket/data.tar.gz",
+    sha256 = "cb8c43418acb05e11619478fbb9c0a76827841cc2d1f58ce9b4d7b5093d1c01e",
+)
+
+minio_archive(
+    name = "test_archive_p1_patch",
+    patch_args = ["-p1"],
+    patches = ["@//:patches/p1.patch"],
+    build_file = "//:BUILD.archive",
+    file_path = "local/bucket/data.tar.gz",
+    sha256 = "cb8c43418acb05e11619478fbb9c0a76827841cc2d1f58ce9b4d7b5093d1c01e",
+)
+
+minio_file(
+    name = "test_file",
+    file_path = "local/bucket/file.txt",
+    sha256 = "8e686326c475d56c005bcda41fe457e777894b57b98be9b29603779d3f6d50b6",
+)
+
+http_archive(
+    name = "rules_python",
+    sha256 = "fa532d635f29c038a64c8062724af700c30cf6b31174dd4fac120bc561a1a560",
+    strip_prefix = "rules_python-1.5.1",
+    urls = [
+        "https://github.com/bazel-contrib/rules_python/releases/download/1.5.1/rules_python-1.5.1.tar.gz",
+    ],
+)
+
+load("@rules_python//python:repositories.bzl", "py_repositories")
+
+py_repositories()
+
+http_file(
+    name = "minio_binary",
+    urls = [
+        "https://dl.min.io/server/minio/release/linux-amd64/minio",
```

Comment:
Oof. These aren't versioned. I'll see if they support versioned releases otherwise, will remove the `sha256` :/